pkg/git/git.go

```diff
@@ -151,10 +149,10 @@ func (g *CLIGit) Push() error {
 func (g *CLIGit) Add() error {
 	fmt.Println("adding unstaged changes")
 	args := []string{
-		"--git-dir", filepath.Join(g.Location, ".git"),
-		"--work-tree", g.Location,
+		"--git-dir", filepath.Join(g.Directory, ".git"),
+		"--work-tree", g.Directory,
 		"add",
-		g.Filename,
+		".",
```

Comment:
I don't necessarily like this, but we don't know what files it generated so we kind of need to add anything that is pending after install.